engine/src/main/java/com/datatorrent/stram/api/StramEvent.java

```diff
@@ -454,17 +459,40 @@ public void setErrorMessage(String errorMessage)
       this.errorMessage = errorMessage;
     }
 
+    public String getLogFileName()
+    {
+      return logFileName;
```

Comment:
I had a  question, if we are going to expose getter setters for logFileName and logFileOffset, why we have to use LogFileInfo object in StramEvent?